### ISSUE TITLE
Return of the FTP error- rsync_from_ncbi.pl: unexpected FTP path (new server?) 

### ISSUE THREAD

**Issue by Madeleine-Meyer** [DerrickWood/kraken2#847]
Hello everyone,

I am trying to download the standard Kraken2 database and I'm having issues with the rsync_from_ncbi_.pl file. Here is my terminal input and the resulting output and error:

**Input:**
`root@myWSLterminal:~/kraken2/install_directory# kraken2-build --standard --db Kraken2_standard_database_NCBI/`

**Output:**
`Downloading nucleotide gb accession to taxon map... done.
Downloading nucleotide wgs accession to taxon map... done.
Downloaded accession to taxon map(s)
Downloading taxonomy tree data... done.
Uncompressing taxonomy data... done.
Untarring taxonomy tree data... done.
rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/762/265/GCF_000762265.1_ASM76226v1
root@myWSLterminal:~/kraken2/install_directory#
`

**Comment by superbioJM** [comment]
Hello，I've encountered the same issue as well. Have you found a solution?

**Comment by Madeleine-Meyer** [comment]
> @Madeleine-Meyer , @BioJM How did you install kraken2? From source or using conda?

Hello @ChillarAnand, I used the source code. I haven't found a solution yet but I've been side stepping the issue for the time being by just building my own custom library with only the species I'm interested in.

**Comment by ChillarAnand** [comment]
Are you using any tools to download specific species data? @Madeleine-Meyer

**Comment by Madeleine-Meyer** [comment]
> Are you using any tools to download specific species data? @Madeleine-Meyer

No, I just downloaded what I wanted from GenBank ([https://www.ncbi.nlm.nih.gov/datasets/genome/?taxon=780](url)). My downloads weren't particularly large, so I didn't need to use NCBI's command line tools [https://www.ncbi.nlm.nih.gov/datasets/docs/v2/download-and-install/](url). I have never used their command line tools, but will likely give them a try in the near future.

**Comment by SamCT** [comment]
This is still a significant issue. I am not keen to rewriting a series of interlinked perl scripts. Hoping that the Kraken2 devs can come to a quick fix so we can have an Kraken2 db that is up-to-date with a broad set of refseq species.
